commands/app/migrate.ts

```diff
@@ -122,9 +125,14 @@ export function builder(yargs: Argv): Argv<MigrateAppArgs> {
     },
     'platform-version': {
       type: 'string',
-      choices: validMigrationTargets,
+      choices: [v2023_2, v2025_2],
+      hidden: true,
+      default: v2025_2,
+    },
+    unstable: {
```

Comment:
The idea is that this will only be used for internal testing and shouldn't be used by external users. We will keep this hidden and keep `unstable` out of the list of platform version choices.